From 25dea008cb29813fe18afce6818e52dbefeb0494 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Wed, 19 Oct 2022 11:37:23 +0800
Subject: [PATCH] Revert "Linux 5.10.149"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 09be132bfe3a3075ddf160cc75865370ea35a0aa.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b824bdb0457c..c40acf09ce29 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 149
+SUBLEVEL = 148
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

